soh/soh/SohGui.cpp

```diff
@@ -266,4 +266,8 @@ namespace SohGui {
     void RegisterPopup(std::string title, std::string message, std::string button1, std::string button2, std::function<void()> button1callback, std::function<void()> button2callback) {
         mModalWindow->RegisterPopup(title, message, button1, button2, button1callback, button2callback);
     }
+
+    void ShowRandomizerSettingsMenu() {
+        mRandomizerSettingsWindow->Show();
+    }
```

Comment:
I'm not very happy with this living inside SohGui.cpp but I've tried to extern the window pointer in other places and just couldn't get it to work. If someone has any other options I'm all ears.